modules/gitops-argo-cd-properties.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * configuring-an-openshift-cluster-with-argo-cd.adoc
+// * depoying-an-application-with-argo-cd.adoc
+
+:_content-type: PROCEDURE
+[id="argo-cd-properties_{context}"]
+= Argo CD custom resource properties
+
+The ArgoCD Custom Resource consists of the following properties:
+
+|===
+|**Name** |**Default** |**Description**
+|`ApplicationInstanceLabelKey` |`mycompany.com/appname`|The `metadata.label` key name where Argo CD injects the app name as a tracking label.
+|`ApplicationSet` |[Object]|`ApplicationSet` controller configuration options.
+|`ConfigManagementPlugins`    |[Empty]|Configuration to add a config management plugin.
+|`Controller`    |[Object]|Argo CD Application Controller options.
+|`Dex`    |[Object]|Dex configuration options.
+|`Disable Admin`    |`false` |Disable the admin user.
+|`GATrackingID`    |[Empty]|The google analytics tracking ID to use.
+|`GAAnonymizeusers`    |[`false` |Enable hashed usernames sent to google analytics.
+|`Grafana`    |[Object]|Grafana configurable options.
+|`HA`    |[Object]|High availablity options.
+|`HelpChatURL`    |`https://mycorp.slack.com/argo-cd` |URL for getting chat help, this will typically be your Slack channel for support.
+|`HelpChatText`    |`Chat now!`|The text for getting chat help.
+|`Image`    |`argoproj/argocd` |The container image for all Argo CD components. This overrides the `ARGOCD_IMAGE` environment variable.
+|`Import`    |[Object]|Import configuration options.
+|`Ingress`    |[Object]|Ingress configuration options.
+|`InitialRepositories`    |[Empty]|Initial git repositories to configure Argo CD to use upon creation of the cluster.
+|`RepositoryCredentials`    |[Object]|Git repository credential templates to configure Argo CD to use upon creation of the cluster.
+|`InitialSSHKnownHosts`    |[Default Argo CD Known Hosts]|Initial SSH Known Hosts for Argo CD to use upon creation of the cluster.
+|`KustomizeBuildOptions`    |[Empty]|The build options/parameters to use with `kustomize build`.
+|`OIDCConfig` |[Empty]|The OIDC configuration as an alternative to Dex.
+|`NodePlacement` |[Empty]|The `NodePlacement` configuration can be used to add the `nodeSelector` and the `tolerations`.
+|`Prometheus` |[Object]|Prometheus configuration options.
+|`RBAC` |[Object]|RBAC configuration options.
+|`Redis` |[Object]|Redis configuration options.
+|`ResourceCustomizations` |[Empty]|Customize resource behavior.
+|`ResourceExclusions` |[Object]|The configuration to completely ignore entire classes of resource group/kinds.
+|`ResourceInclusions` |[Object]|The configuration to configure which resource group/kinds are applied.
+|`Server` |[Object]|Argo CD Server configuration options.
+|`SSO` |[Object]|Single sign-on options.
+|`StatusBadgeEnabled` |`ture` |Enable application status badge.
+|`TLS` |[Object]|TLS configuration options.
+|`UserAnonyousEnabled` |`true` |Enable anonymous user access.
```

Comment:
```suggestion
|`UserAnonymousEnabled` |`true` |Enable anonymous user access.
```

Reply:
@iam-veeramalla please suggest.

Reply:
@DebarghoGhosh actually it should be `UsersAnonymousEnabled` not both :p

https://github.com/argoproj-labs/argocd-operator/blob/808d0a200f96fc403731f9887ad5818ce58988f8/api/v1alpha1/argocd_types.go#L632